packages/gatsby-transformer-xml/src/gatsby-node.js

```diff
@@ -2,7 +2,7 @@ const parseXml = require(`xml-parser`)
 const crypto = require(`crypto`)
```

Comment:
please remove this import ( https://circleci.com/gh/gatsbyjs/gatsby/6659?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link )

Reply:
my bad